tests/unit/info.tcl

```diff
@@ -349,7 +349,9 @@ start_server {tags {"info" "external:skip"}} {
             set value [s instantaneous_eventloop_duration_usec]
             if {$::verbose} { puts "instantaneous metrics instantaneous_eventloop_duration_usec: $value" }
             assert_morethan $value 0
-            assert_lessthan $value [expr $retries*22000] ;# default hz is 10, so duration < 1000 / 10, allow some tolerance
+            if {!$::tsan} {
```

Comment:
what about tag these tests as asan:skip, then skip then when ::tsan is true?

Reply:
Great idea, added tsan:skip tag to skip tests that fail due to timing issues with thread sanitizer.